api/host_query_db.py

```diff
@@ -194,6 +194,23 @@ def get_host_tags_list_by_id_list(
     return host_tags_dict, len(host_id_list)
 
 
+def _add_tag_values(host_namespace, tag_key, tag_value, host_id, host_tags, host_tags_tracker) -> Tuple[dict, dict]:
+    host_tag_str = (
+        f"{_convert_null_string(host_namespace)}/{_convert_null_string(tag_key)}={_convert_null_string(tag_value)}"
+    )
```

Comment:
```suggestion
    host_tag_str = (
        f"{_convert_null_string(host_namespace)}/{_convert_null_string(tag_key)}={_convert_null_string(tag_value)}"
    )
    if not _convert_null_string(tag_value):
        host_tag_str = (
            f"{_convert_null_string(host_namespace)}/{_convert_null_string(tag_key)}="
        )
```

If the value here is a null or [] then the `host_tag_str` would end with `=None` which wouldn't be good for the regex search.

Reply:
Oops, brainfart. For some reason that equated to blank. Fixing

Reply:
OK, fixed. I just modified the value used in the `host_tag_str`